content/en/docs/release-notes/rn-armory-operator/armory-operator-v1-2-6.md

```diff
@@ -0,0 +1,19 @@
+---
+title: v1.2.6 Armory Operator
+toc_hide: true
+version: 01.02.06
+description: Release notes for Armory Operator v1.2.6
+---
+
+## 05/07/2021 Release Notes
```

Comment:
And probably include that note about registries in this release as well?

Reply:
taken care of